examples/mpi/lammps/test.bats

```diff
@@ -79,15 +79,20 @@ lammps_try () {
 @test "${ch_tag}/friction" { lammps_try friction; }
 @test "${ch_tag}/melt"     { lammps_try melt; }
 
-# This test busy-hangs after several:
+# Previously running the python example in /examples/python resulted in a busy
+# hang with the error:
 #
 #   FOO error: local variable 'foo' referenced before assignment
 #   Inside simple function
 #
-# Perhaps related to --join?
+# Using mpi4py to launch the examples in /python/examples does not appear to
+# exhibit this behavior so using simple.py instead
```

Comment:
I think you can probably skip explaining the previous behavior.

Reply:
I'll remove.